src/utils/api.ts

```diff
@@ -0,0 +1,115 @@
+import config from '../config';
+import constants from '../constants';
+import axios from 'axios';
+import { IMessageTransaction, ITransactionData, IPostTransactionResponse } from '../interfaces';
+
+const path = (endpoint: string): string => config.nodes[0] + '/api' + endpoint;
+
+export const checkAccountExists = async (id: string): Promise<boolean> => {
+  const hits = await axios
+    .get(path('/delegates/' + id))
+    .then(res => Object.keys(res.data.data).length)
```

Comment:
This is inconsistent. You are using `await` but then also `then` and `catch` instead of simply using `await` and `try/catch`.

Reply:
Resolved in https://github.com/ArkEcosystem/poc-ark-messenger/pull/5